libs/@local/hash-isomorphic-utils/src/simplify-properties.ts

```diff
@@ -38,11 +38,17 @@ export type Simplified<T extends Entity | BpEntity> = Omit<T, "properties"> & {
 export const simplifyProperties = <T extends EntityPropertiesObject>(
   properties: T,
 ): SimpleProperties<T> => {
-  return typedEntries(properties).reduce(
-    (acc, [key, value]) => ({
+  // @ todo: this makes additional assumptions about the structure of the key.
+  //  This should be done more consistently, like through a branded type that
+  //  introduces well-known URL structures (this approach is used turbine)
```

Comment:
okay, I thought this was also called on external types, that may not be of HASH origin. I just found this because I was switching from `lodash` to `lodash/fp/camelCase`, which has stricter typing (`string` instead of `string|undefined`).

Reply:
I am still not a fan that this assumption is carried through the codebase implicitly.